controllers/user.js

```diff
@@ -131,4 +136,102 @@ export default class UserController {
       });
     });
   }
+
+  /**
+   * @description reset user password
+   * @param {object} req http request object
+   * @param {object} res http response object
+   * @returns {object} response
+   */
+  static async resetPassword(req, res) {
+    const { email } = req.body;
+    const passwordResetToken = shortId.generate();
+    try {
+      // save password reset token to db
+      await User.update(
+        { passwordResetToken },
```

Comment:
@youngestdj I mentioned that this update query should be done at once and not in two different places

Reply:
I have used a single update query

Reply:
@youngestdj I can't see that here. There's another update on line 157. That's what I'm referring to